package.json

```
@@ -10,20 +10,43 @@
     "start": "node ./build/index.js",
     "debug": "ts-node ./src/index.ts",
     "compile": "tsc -p .",
+    "format": "prettier --write",
+    "lint": "eslint --fix",
+    "fix": "npm run format && npm run lint",
```

`lint` で修正モードにするのはちょっと不便
```suggestion
    "lint": "eslint",
    "fix": "npm run format && npm run lint -- --fix",
```